tutorials/simulators/DualSPHysics.md

```diff
@@ -276,15 +276,7 @@ Now, we can run this script, and the output should look something like this:
 
 ■ Tier: Power-User
 
-■ Credits
-
-  Power-User (tier)               0.00 USD
-  ----------------------------------------
-  Total                         989.99 USD
-
-■ Active Campaigns
-
- Not currently enrolled in any campaign.
+■ Credits: 1000.00 US$
```

Comment:
I believe the dollar sign should appear before

Reply:
I don't know what's intended, but the current format is `US$`.

```
$ inductiva user info
...

■ Tier: Power-User

■ Credits: 9982.08 US$

...
```